src/Components/CreateImageWizardV2/steps/FileSystem/FileSystemConfiguration.tsx

```diff
@@ -202,7 +202,6 @@ const FileSystemConfiguration = () => {
       <TextContent>
         <Button
           ouiaId="add-partition"
-          data-testid="file-system-add-partition"
```

Comment:
These test ids are used in unit tests so we can keep them as they are. I think a rule of thumb we could follow is to freely use test ids for our testing purposes and preferably using ouiaIds when it comes to integration testing and pr_check. Which probably means a lot of components will have both :sweat_smile: